src/dynarec/arm64/dynarec_arm64_67_avx.c

```diff
@@ -126,6 +126,20 @@ uintptr_t dynarec64_67_AVX(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int
                 }
                 break;
 
+            case 0x6E:
+                INST_NAME("VMOVD Gx, Ed");
+                nextop = F8;
+                GETGX_empty(v0);
+                GETED(0);
+                if(rex.w) {
+                    FMOVDx(v0, ed);
+                } else {
+                    VEORQ(v0, v0, v0); // RAZ vector
```

Comment:
Why not `FMOVSw(...)` here if `FMOVDx(...)` is good enough for `rex.w` case?

Reply:
You need the `VEORQ(...)` on both branch of the if actually, because you are setting `Gx` here and not `Gm`.

I suggest you use FMOV or VMOVQ on both branch also. Mayby favor `VMOVQ` has it's a `Gx` here...